iosTraining-Reeen/ModalHolderViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  EmptyViewController.swift
+//  iosTraining-Reeen
+//
+//  Created by 高橋 蓮 on 2023/06/09.
+//
+
+import UIKit
+
+final class ModalHolderViewController: UIViewController {
+    
+    var heldViewController: UIViewController?
+    
+    override func viewDidAppear(_ animated: Bool) {
+        guard let nextViewController = heldViewController else { return }
```

Comment:
[imo]
好みの問題ですが、今のSwiftはこの場合省略もできるので、手抜きできるものは積極手抜きしていいかとw
```suggestion
        guard let heldViewController else { return }
```